src/robot/libraries/String.py

```diff
@@ -363,15 +364,16 @@ def get_lines_matching_regexp(self, string, pattern, partial_match=False):
         | ${lines} = | Get Lines Matching Regexp | ${result} | Reg\\\\w{3} example |
         | ${lines} = | Get Lines Matching Regexp | ${result} | Reg\\\\w{3} example | partial_match=true |
         | ${ret} =   | Get Lines Matching Regexp | ${ret}    | (?i)FAIL: .* |
+        | ${ret} =   | Get Lines Matching Regexp | ${ret}    | FAIL: .* | flags=IGNORECASE
```

Comment:
`|` missing at the end. It probably actually doesn't matter, but it should be used for consistence as above lines have it.